docs/dcomibmtoolsattachretry.md

```diff
@@ -0,0 +1,46 @@
+<!--
+* Copyright (c) 2017, 2024 IBM Corp. and others
```

Comment:
For a new file, should it be `2024, 2024`?

Reply:
No, as per the guidelines specified in https://github.com/eclipse-openj9/openj9-docs/blob/master/CONTRIBUTING.md#style-guidelines, the dates refer to the complete documentation set, not to individual files and so should be the same in all topics.